src/test/resources/TestNG.xml

```diff
@@ -5,10 +5,19 @@
 
     <test name="Example Test" preserve-order="false">
         <classes>
+<<<<<<< Updated upstream
             <class name="ChangeUsernameTests"/>
 <!--            <class name="BaseTest"/>-->
 <!--            <class name="LoginTests"/>-->
 <!--            <class name="RegistrationTests"/>-->
+=======
+            <class name="LoginTests"/>
+            <class name="BaseTest"/>
+            <class name="HW14"/>
+            <class name="HW15"/>
+            <class name="HW16"/>
+            <class name="HW17"></class>
```

Comment:
To avoid conflicts and run tests in the HW17 branch, I suggest this option:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNG">

  <test name="Example Test" preserve-order="false">
    <classes>
      <class name="LoginTests"/>
      <class name="BaseTest"/>
      <class name="HW16"/>
      <class name="RegistrationTests"/>
      <class name="ChangeUsernameTests"/>
      <class name="HW17"/>
    </classes>
  </test>
</suite>
```